simplex/messages.go

```diff
@@ -0,0 +1,233 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package simplex
+
+import (
+	"github.com/ava-labs/simplex"
+
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/proto/pb/p2p"
+)
+
+func newP2PSimplexBlockProposal(
```

Comment:
Everywhere that we have `newP2PSimplex...` should probably be replaced with `new`, we are in the `simplex` package, everything is simplex related.